components/hero/hero.snip.html

```diff
@@ -0,0 +1,20 @@
+{{#hero}}
+  <div class="ampstart-hero center">
+    <div class="ampstart-hero-img">
+      {{#images}}
+        {{#image}}
+          <amp-img src="{{url}}" width="{{width}}" height="{{height}}" layout="responsive" media="{{media}}" alt="{{alt}}"></amp-img>
```

Comment:
you should use an SRC set to point to normal and 2x images for retina support, also look at the sizes attribute https://www.ampproject.org/docs/guides/responsive/art_direction , this could also be of interest to you